docs/basics/overview.md

```diff
@@ -0,0 +1,232 @@
+
+**Models** are the neural network architectures that learn to approximate posterior distributions. They are standard Flax NNX modules.
+
+GenSBI provides three main model architectures:
+
+- **Flux1**: A double-stream transformer using Rotary Position Embeddings (RoPE). Best for high-dimensional problems.
+- **Simformer**: A single-stream transformer that explicitly embeds variable IDs. Best for low-dimensional problems.
+- **Flux1Joint**: A single-stream variant of Flux1 for explicit joint modeling. Good for likelihood-dominated problems.
+
+**Example:**
+```python
+from gensbi.models.flux1 import Flux1, Flux1Params
+from flax import nnx
+
+params = Flux1Params(
+    in_channels=1,
+    num_heads=8,
+    depth=12,
+    depth_single_blocks=24,
+    axes_dim=[obs_dim],
+    rngs=nnx.Rngs(0),
+    obs_dim=3,
+    cond_dim=5,
+)
+
+model = Flux1(params)
+```
+
+### 2. Model Wrappers
+
+**Model Wrappers** provide a standard interface for models to be used by ODE/SDE solvers during sampling. They standardize how models are called and provide methods for computing the vector field and divergence needed for numerical integration.
+
+Three types of wrappers exist:
+
+- **Unconditional**: For unconditional density estimation
+- **Conditional**: For conditional inference (standard SBI: estimate θ given x)
+- **Joint**: For joint inference (estimate multiple variables simultaneously)
+
+The wrapper provides:
+- Standardized calling interface for solvers
+- `get_vector_field()` method for ODE integration
+- `get_divergence()` method when needed for likelihood computation
+
+**Note**: Wrappers are only used during sampling/inference. During training, the unwrapped model is called directly.
+
+### 3. Recipes and Pipelines
+
+**Recipes** define complete end-to-end procedures for a specific task (e.g., SBI, VAE training). **Pipelines** are specific implementations of these recipes using particular generative modeling approaches (e.g., flow matching or diffusion).
+
+Currently, GenSBI provides two main recipes:
+- **SBI Recipe**: For simulation-based inference
+- **VAE Recipe**: For training variational autoencoders
+
+**Pipelines** handle all aspects of training and inference:
+
+- Data loading and batching
+- Training loop (optimizer, learning rate scheduling, early stopping)
+- Validation and checkpointing
+- Exponential Moving Average (EMA) of weights
+- Model wrapping for sampling
+
+**Key SBI Pipelines:**
+- `Flux1FlowPipeline`: Flow matching with Flux1 model
+- `SimformerFlowPipeline`: Flow matching with Simformer model
+- `Flux1JointFlowPipeline`: Flow matching with Flux1Joint model
+- Similar diffusion variants exist
+
+**Example:**
+```python
+from gensbi.recipes import Flux1FlowPipeline
+
+pipeline = Flux1FlowPipeline(
+    train_dataset=train_iter,
+    val_dataset=val_iter,
+    obs_dim=3,
+    cond_dim=5,
+    params=flux1_params,
+)
+
+# Train
+pipeline.train(rngs=nnx.Rngs(0))
+
+# Sample (note: x_o is the observed data, not 'condition')
```

Comment:
x_o is the observed measurement data, which is used to condition the density estimation sampling procedure in order to obtain samples from the posterior distribution p(theta|x_o)